data/json/items/comestibles/egg.json

```diff
@@ -637,5 +637,117 @@
     "volume": "650 ml",
     "description": "A large clump of the eggs of a frog.  The eggs themselves are gelatinous and transparent.",
     "rot_spawn": "GROUP_EGG_FROG"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_goose",
+    "name": "goose egg",
+    "copy-from": "egg_chicken",
+    "calories": 266,
+    "volume": "125 ml",
+    "weight": "144 g",
+    "vitamins": [ [ "calcium", 9 ], [ "iron", 29 ] ],
+    "rot_spawn": "GROUP_EGG_GOOSE"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_goose_golden",
+    "name": "golden goose egg",
+    "copy-from": "egg_goose",
+    "calories": 300,
+    "volume": "125 ml",
+    "weight": "155 g",
+    "vitamins": [ [ "calcium", 9 ], [ "iron", 29 ] ],
+    "rot_spawn": "GROUP_EGG_GOOSE"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_hummingbird",
+    "name": "hummingbird egg",
+    "copy-from": "egg_bird",
```

Comment:
Hummingbird eggs need to be way smaller than the base one.

Reply:
changed the volume and weight of the eggs to match the biggest approximate to the real deal